tests/test_geovector.py

```diff
@@ -29,3 +29,11 @@ def test_query(self):
         assert vector2 is not self.glacier_outlines
 
         assert vector2.ds.shape[0] < self.glacier_outlines.ds.shape[0]
+
+    def test_bounds(self):
+
+        bounds = self.glacier_outlines.bounds
+
+        assert bounds.left < bounds.right
```

Comment:
Shouldn't we also test it is equal to either a hard-coded value, or to geopandas bounds?

Reply:
Added check to geopandas bounds in 78fe3b2